modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -130,14 +130,14 @@ parameters:
 +
 [source,terminal]
 ----
-$ oc process -f <templatefile> | oc apply -f -
+$ oc process -f <templatefile> | oc apply -n openshift-logging -f -
 ----
 +
 For example:
 +
 [source,terminal]
 ----
-$ oc process -f eventrouter.yaml | oc apply -f -
+$ oc process -f <templatefile> | oc apply -n openshift-logging -f -
```

Comment:
This line:

 $ oc process -f <templatefile> | oc apply -n openshift-logging -f -

it should be:

  $ oc process -f eventrouter.yaml | oc apply -n openshift-logging -f -